src/Validator.php

```diff
@@ -170,7 +180,7 @@ private function validateInternal($value, iterable $rules, ValidationContext $co
     }
 
     /**
-     * @param array $rules
+     * @param iterable<Closure|RuleInterface> $rules
```

Comment:
```suggestion
     * @param iterable $rules
```

Reply:
I've removed it completely because it just matches the type hint and no description is present.